### ISSUE TITLE
[MOBO] - Add alternative footprint for vacuum sensor

### ISSUE THREAD

**Issue by wilhelmzeuschner** [sphawes/index#164]
Currently the [2SMPP03](https://omronfs.omron.com/en_US/ecb/products/pdf/en-2smpp-03.pdf) is used:
![image](https://user-images.githubusercontent.com/37621765/108627678-295efd80-7457-11eb-856e-8ec6d874aaf2.png)
![image](https://user-images.githubusercontent.com/37621765/108627657-10564c80-7457-11eb-8252-148214c09dcb.png)
![image](https://user-images.githubusercontent.com/37621765/108627803-b0ac7100-7457-11eb-8733-34840fa2b5b2.png)

It has footprint a that differs from other cheaper vacuum sensors available. 

For higher flexibility in sourcing parts I suggest to add a footprint for a sensor with this footprint:
![image](https://user-images.githubusercontent.com/37621765/108627639-f583d800-7456-11eb-97e9-5198ea002a6f.png)
Pinout:
![image](https://user-images.githubusercontent.com/37621765/108627559-91611400-7456-11eb-8e57-83cbf162c698.png)
Here are two examples for this type of sensor:
[XGZP Pressure Sensor series](http://m.eleparts.co.kr/data/goods_old/data/XGZP_SOP6.pdf) 
![image](https://user-images.githubusercontent.com/37621765/108627729-5b705f80-7457-11eb-8e8f-24a52ad276b0.png)
or [MPS3117-006GC-A2](https://datasheet.lcsc.com/szlcsc/Metrodyne-Microsystem-MPS-3118-015GC_C114719.pdf) <- Note that the last sensor is not for vacuum sensing applications! But the footprint / pinout is the same. This type of sensor seems to be very common.

I bought the XGZP sensor. Here is an [overview ](https://www.sgbotic.com/products/datasheets/sensors/02976-datasheet.pdf)of the different models: 
![image](https://user-images.githubusercontent.com/37621765/108627879-03862880-7458-11eb-9037-f82d9011355d.png)
Here's a [PCB](https://www.aliexpress.com/item/32923552711.html)  that should have a -40kPa sensor.
Supposedly -100kPa: [XGZP101SB1 ](https://www.aliexpress.com/item/32709990616.html) 

AliExpress certainly is not a reliable source to get parts. But the flexibility to have this common type of sensor-footprint on the PCB would come in handy. The two footprints could be superimposed, the alternative is on the top layer and the current sensor is on the bottom layer, facing up through the hole.

**Comment by G-Pereira** [comment]
Nice work! This makes a lot of sense, we indeed have the possibility to add both footprints.

**Comment by G-Pereira** [comment]
Hey again! I just checked this in more detail and saw that LCSC doesn't have any vacuum sensors. This footprint would just fit the AliExpress sensors which we will not be able to support and thus the gain of the amplifier should be set by the user as well as the current source parameters. If one goes with Mouser or Digikey they should get our recommended sensor, the 2SMPP03.
Anyhow, I've added this in!
